doc/reference/ufunc.rst

```diff
@@ -6,6 +6,50 @@ Universal Functions (ufunc)
 .. https://docs.scipy.org/doc/numpy/reference/ufuncs.html
 
 DPNP provides universal functions (a.k.a. ufuncs) to support various element-wise operations.
+DPNP ufunc supports following features of NumPy’s one:
```

Comment:
It would be better to reuse numpy like description here with a reference to numpy documentation at the end.